ui/summary_test.go

```diff
@@ -160,7 +162,96 @@ func createTestMetrics() map[string]*stats.Metric {
 	metrics["vus"] = gaugeMetric
 	metrics["http_reqs"] = countMetric
 	metrics["checks"] = checksMetric
-	metrics["my_trend"] = &stats.Metric{Name: "my_trend", Type: stats.Trend, Contains: stats.Time, Sink: sink}
+	metrics["my_trend"] = &stats.Metric{
+		Name:     "my_trend",
+		Type:     stats.Trend,
+		Contains: stats.Time,
+		Sink:     sink,
+		Thresholds: stats.Thresholds{
```

Comment:
Can you add `Tainted` and set it so that it shows in the `TestSummary` (and obviously fix it as well ;))

Reply:
I added `Tainted`, nut `TestSummary` still passes, not sure what to fix here.

Reply:
Ah, it does fail